File: 6kyu/squared_spiral_1/example_test_cases.py
import codewars_test as test


@test.describe("Tests")
def tests():
    @test.it("Fixed Tests")
    def fixed_tests():
        test.assert_equals(squared_spiral(0), (0, 0))
        test.assert_equals(squared_spiral(6), (-1, -1))
        test.assert_equals(squared_spiral(10), (2, 0))
        test.assert_equals(squared_spiral(12), (2, 2))
        test.assert_equals(squared_spiral(30), (3, 3))

File: 5kyu/squared_spiral_1/javascript/example_test_cases.js
const chai = require("chai");
const assert = chai.assert;

describe("Tests", function () {
  it("Fixed Tests", function () {
    assert.deepEqual(squaredSpiral(0), [0, 0]);
    assert.deepEqual(squaredSpiral(6), [-1, -1]);
    assert.deepEqual(squaredSpiral(10), [2, 0]);
    assert.deepEqual(squaredSpiral(12), [2, 2]);
    assert.deepEqual(squaredSpiral(30), [3, 3]);
  });
});
